CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Removed
+- `getTransactionStatus` is deprecated. Please use `getPaymentStatus` instead.
```

Comment:
was deprecated in `1.6.0`, and has now been removed?

I get this is under a `Removed` header, but `deprecated` implies to me that it's still there.

Reply:
Done. I think i just copy and pasted the wrong header.